tei/la-estatua-de-prometeo.xml

```diff
@@ -6046,11 +6158,12 @@
             <l>del juvenil escuadrón</l>
             <l>de las zagalas del valle</l>
             <l>y del popular rumor</l>
-            <l>del demás pueblo, diciendo Bosque y risco.</l>
+            <l>del demás pueblo, diciendo</l>
+            <stage>Bosque y risco.</stage>
```

Comment:
Again, not sure this is the right word. Though, in the manuscript, both instances of "bosque" seem to be the same word. Cf. https://www.cervantesvirtual.com/nd/ark:/59851/bmcb2849